src/btree.c

```diff
@@ -1512,7 +1499,7 @@ btree_defragment_or_split_child_index(cache              *cc,
                                       btree_node         *parent,
                                       uint64      index_of_child_in_parent,
                                       btree_node *child,
-                                      const slice key_to_be_inserted,
+                                      key         key_to_be_inserted,
```

Comment:
`const` is kinda meaningless on slices passed-by-value, since the pointer within the slice is already const.  It was my mistake to put const in in the first place.  So I've just been removing them as I go.